base.yaml

```diff
@@ -267,3 +267,14 @@ radar_upload_source_connector:
   _install: true
   replicaCount: 1
   client_id: radar_upload_connect
+
+
+# --------------------------------------------------------- 20-s3-connector.yaml ---------------------------------------------------------
```

Comment:
Would it make sense to change the `20-hdfs.yaml` to `20-output.yaml` and put the s3 option in there as well?

Reply:
We can. But I was planning to do that on a separate PR. Technically we can run both connectors in parallel and adding s3 connector is mainly for intermediate data.

Reply:
I understand, we can decide on that later then